content/courses/program-security/type-cosplay.md

```diff
@@ -2,69 +2,71 @@
 title: Type Cosplay
 objectives:
   - Explain the security risks associated with not checking account types
-  - Implement an account type discriminator using long-form Rust
+  - Implement an account type discriminator using native Rust
   - Use Anchor's `init` constraint to initialize accounts
   - Use Anchor's `Account` type for account validation
 description:
-  "Under the risks of accounts of the wrong type being used in instruction, and
-  use account type checks to mitigate them."
+  "Understand the risks of using incorrect account types in instructions and how
+  to mitigate them with account type checks."
 ---
 
 ## Summary
 
-- Use discriminators to distinguish between different account types
-- To implement a discriminator in Rust, include a field in the account struct to
-  represent the account type
+- **Use discriminators** to distinguish between different account types.
```

Comment:
We should add:

```
- Discriminators are data written to account that determines the type of data stored inside
```

or something similar to make sure people know what they are.

Reply:
Yes adding it. Also, should I add a section to explain the concept?

Reply:
Added a bit of explanation in the same line.

Reply:
Thanks, update is clear.